docs/application/dotnet/guides/nui/text.md

```diff
@@ -374,17 +325,48 @@ label.Text = "&#9786; &#x263a;"; //smile-face
 
 ```
 
+
+### TextLabel properties

```

Comment:
### TextLabel properties

Only here P is in lower case, all other places  in the headings P seem to be in upper case. Request to check for consistency.